OWNERS

```diff
@@ -8,8 +8,7 @@
       owners: [
         {name: 'cramforce', requestReviews: false},
         {name: 'dvoytenko', requestReviews: false},
-        {name: 'jridgewell', requestReviews: false},
-        {name: 'kristoferbaxter', requestReviews: false},
+        {name: 'jridgewell', requestReviews: false}
```

Comment:
I think it's worth adding someone else who actively works on AMP so there's more than one top-level reviewer. Maybe @mrjoro?

Reply:
FWIW Joey has owners approval over all OWNERS files currently

Reply:
Right, but that doesn't enable Joey to provide a top-level approval for a PR that touches a lot of source files, e.g. a large refactor. Figured it would be worth having more than just Justin available for those kinds of approvals. (Unless owners of `OWNERS` files also own the directory contents?)

Reply:
That's true, but I would maybe argue that someone like you or @alanorozco (who has intimate knowledge of runtime, infra, and component code) would be a better candidate for that role given the need to assess large-scale refactors, which doesn't seem like Joey's wheelhouse as much these days.

Reply:
Fair point. @mrjoro WDYT? Happy to take this on, since this will just be an owners review that prevents mistaken merging of breaking code. More specific WG member / owner reviews will still be required for most PRs.

Reply:
SGTM